packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -0,0 +1,172 @@
+	const projectPath = exampleDir + '/not-theme';
+	expect(isThemeDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+test('isThemeDirectory returns false for a directory with style.css but without Theme Name', () => {
+	const projectPath = exampleDir + '/not-theme';
+
+	expect(isThemeDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+// WP_CONTENT mode
+test('isWpContentDirectory detects a WordPress wp-content directory and infer WP_CONTENT mode', () => {
+	const projectPath = exampleDir + '/wp-content';
+	expect(isWpContentDirectory(projectPath)).toBe(true);
+	expect(isWordPressDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.WP_CONTENT);
+});
+
+test('isWpContentDirectory returns false for wp-content parent directory', () => {
+	const projectPath = exampleDir + '/index';
+	expect(isWpContentDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+test.skip('isWpContentDirectory returns false for a directory with only one directory of plugins or themes', () => {
+	const projectPath = exampleDir + '/not-wp-content';
+	expect(isWpContentDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+// WordPress mode
+test('isWordPressDirectory detects a WordPress directory and WORDPRESS mode', () => {
+	const projectPath = exampleDir + '/wordpress';
+
+	expect(isWordPressDirectory(projectPath)).toBe(true);
+	expect(inferMode(projectPath)).toBe(WPNowMode.WORDPRESS);
+});
+
+test('isWordPressDirectory returns false for non-WordPress directory', () => {
+	const projectPath = exampleDir + '/nothing';
+
+	expect(isWordPressDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+// WordPress developer mode
+test.skip('isWordPressDevelopDirectory detects a WordPress-develop directory and WORDPRESS_DEVELOP mode', () => {
+	const projectPath = __dirname;
+	jest.spyOn(fs, 'existsSync').mockImplementation((fileOrFolder) => {
+		const existingFiles = [
+			'src',
+			'src/wp-content',
+			'src/wp-includes',
+			'src/wp-load.php',
+			'build',
+			'build/wp-content',
+			'build/wp-includes',
+			'build/wp-load.php',
+		].map((file) => path.join(projectPath, file));
+		return existingFiles.includes(fileOrFolder as string);
+	});
+
+	expect(isWordPressDevelopDirectory(projectPath)).toBe(true);
+	expect(inferMode(projectPath)).toBe(WPNowMode.WORDPRESS_DEVELOP);
+});
+
+test('isWordPressDevelopDirectory returns false for non-WordPress-develop directory', () => {
+	const projectPath = exampleDir + '/nothing';
+
+	expect(isWordPressDevelopDirectory(projectPath)).toBe(false);
+	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
+});
+
+test.skip('isWordPressDevelopDirectory returns false for incomplete WordPress-develop directory', () => {
```

Comment:
Let's adjust or remove those tests instead of skipping them @sejas @danielbachhuber

Reply:
@adamziel @sejas Yep, I will adjust them and get them running again.

Reply:
https://github.com/WordPress/wordpress-playground/pull/333